src/main/java/seedu/address/model/student/admin/ClassTime.java

```diff
@@ -96,8 +104,9 @@ public String convertClassTimeToUserInputString() {
 
     @Override
     public String toString() {
+        String dayDisplayName = dayOfWeek.getDisplayName(TextStyle.FULL, Locale.ENGLISH);
         return String.format("Day of week: %s, Start time: %s, End Time: %s",
-                this.dayOfWeek,
+                dayDisplayName,
                 this.startTime.format(TIME_FORMATTER),
                 this.endTime.format(TIME_FORMATTER));
```

Comment:
Need to change `TIME_FORMATTER` to `OUTPUT` for both startTime and endTime

Reply:
Okay thanks!